src/include/api.h

```diff
@@ -189,6 +189,8 @@
     API_END(s, 0);                       \
     return ((ret) == WT_NOTFOUND ? ENOENT : (ret))
 
+#define API_FIRST_ENTRY(s) (s)->api_call_counter == 1
```

Comment:
This deserves a comment. I prefer "user" to "first", I think, but it's still not super clear.